tfchain_bridge/pkg/bridge/bridge.go

```diff
@@ -458,6 +487,31 @@ func (bridge *Bridge) submitBurnTransaction(ctx context.Context, burnReadyEvent
 	return bridge.subClient.SetBurnTransactionExecuted(bridge.identity, uint64(burnReadyEvent.BurnTransactionID))
 }
 
+func (bridge *Bridge) handleMint(amount *big.Int, target substrate.AccountID, mintID string) error {
+	// TODO check if we already minted for this txid
+	minted, err := bridge.subClient.IsMintedAlready(bridge.identity, mintID)
+	if err != nil && err != substrate.ErrMintTransactionNotFound {
+		return err
+	}
+
+	if minted {
+		log.Error().Msgf("transaction with id %s is already minted", mintID)
```

Comment:
This shouldn't be an error, if it is not expected to happen make it a warning, otherwise info